src/stories/Steps.stories.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from '@storybook/addon-a11y';
+import { IntlProvider } from 'react-intl';
+
+
+import Steps from 'js/modules/Steps';
+import jsonFileData from '__tmpdata/pages';
+
+const stepsHtml = `
+  <ul>
+    <li><p>Check the “What do I do with” tool below to find out what items are accepted. Some items are free to drop off and some items, like tires, require a fee.</p></li>
+    <li><p>Review the Household Hazardous Waste Do’s and Don’ts below.</p></li>
+    <li><p>Drop off your items at the Recycle and ReUse Center at 2514 Business Center Dr, Austin, TX 78744.</p></li>
+  </ul>
+`
+
+storiesOf('Steps', module)
+  .addDecorator(checkA11y)
+  .add('Steps', () => (
+    <IntlProvider locale="en">
+      <div className="wrapper container-fluid">
+        <Steps steps={stepsHtml} />
```

Comment:
@ifsimicoded, what would you think about changing the name of this prop to be `stepsHtml`, `stepsUnorderedListHtml` or something like that to signal to users of the component what the Step expects?

Reply:
This makes a lot of sense. It's my understanding that this was the simplest way for wagtail to send janis data. so the hope is we'll just be looping at some point over raw content and not dumping mark up. But maybe until that change happens, it makes a lot more sense to rename this to be really clear. Maybe even a comment in the component that shows the expected data format.

Reply:
stepsHtmlFromAdmin?